### ISSUE TITLE
How to open graph in a Graphvizer Viewer window?

### ISSUE THREAD

**Issue by pancodia** [hao-lee/Graphvizer#22]
I installed Graphvizer Viewer and could open by running `python GraphvizerViewer.py`. However, how can I open png in the GraphvizerViewer window?

**Comment by pancodia** [comment]
Oh. I can drag the png file to the Viewer's window. But can I open the graph directly from Sublime?

**Comment by hao-lee** [comment]
Currently this plugin can't invoke external viewer directly but this feature has been added to todo list. I will find an elegant way to integrate them.

**Comment by gerph** [comment]
I found it easiest just to split the window vertically (with Layout->rows:2) and then in the lower window select the generated PNG. Then it updates as the image is generated.

**Comment by hao-lee** [comment]
@gerph Yeah, viewing image in Sublime is convenient if thie image is not very large. Otherwise, a more powerful image viewer is needed.
